src/ui-kit/experimental/hierarchical/hierarchical-tree-grid/hierarchical-tree-grid.component.html

```diff
@@ -1,21 +1,16 @@
-<div class="example-container usa-grid">
-  <ng-container *ngTemplateOutlet="template"></ng-container>
+<div class="example-container usa-grid" style="max-height: 400px; overflow-y: scroll;">
```

Comment:
@yerramshilpa Can you move this into the styles on the component? @carlosvalle can help you move it into the CSS structure later.